website/docs/r/cloud_backup_schedule.html.markdown

```diff
@@ -0,0 +1,196 @@
+---
+layout: "mongodbatlas"
+page_title: "MongoDB Atlas: cloud_backup_schedule"
+sidebar_current: "docs-mongodbatlas-resource-cloud-backup-schedule"
+description: |-
+    Provides a Cloud Backup Schedule resource.
+---
+
+# mongodbatlas_cloud_backup_schedule
+
+`mongodbatlas_cloud_backup_schedule` provides a cloud backup schedule resource. The resource lets you create, read, update and delete a cloud backup schedule.
+
+-> **NOTE:** Groups and projects are synonymous terms. You may find `groupId` in the official documentation.
+
+In the Terraform MongoDB Atlas Provider 1.0.0 we have re-architected the way in which Cloud Backup Policies are manged with Terraform to significantly reduce the complexity. Due to this change we've provided multiple examples below to help express how this new resource functions.
+
+
+~> **IMPORTANT:**   `policies.0.policy_item.#.id` is obtained when the cluster for which the policy applies is created and has `cloud_backup` is set to true.
```

Comment:
My suggestion had an extra word in it, can you remove "is".  Small issue but it reads oddly.